src/audio/tdfb/tdfb_direction.c

```diff
@@ -125,10 +125,8 @@ static int16_t max_mic_distance(struct tdfb_comp_data *cd)
 
```

Comment:
Remove from commit text "Optimizations:
- Enhance `theoretical_time_differences` efficiency by substituting
  division with multiplication using a precomputed reciprocal.". That patch is elsewhere.